packages/git/src/browser/dirty-diff/dirty-diff-preferences.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { interfaces } from 'inversify';
+import {
+    createPreferenceProxy,
+    PreferenceProxy,
+    PreferenceService,
+    PreferenceContribution,
+    PreferenceSchema
+} from '@theia/core/lib/browser';
+
+export const enum DIRTYDIFF {
```

Comment:
we don't need such enums, preference proxies provide typed interfaces, you cannot use any other properties, you will get a compiler error

Reply:
ok